rtos/rtx5/TARGET_CORTEX_M/rtx_lib.c

```diff
@@ -595,7 +595,6 @@ void *__user_perthread_libspace (void) {
 typedef void *mutex;
 
 // Initialize mutex
-__USED
```

Comment:
When that was in, it was conflicting with our define. We should look into why Used + Weak was not working correctly.